src/app/global/local_account_sensitive_settings.nim

```diff
@@ -96,6 +96,8 @@ const LSS_KEY_IS_DDMMYY_DATE_FORMAT* = "is_DDMMYY_date_format"
 const DEFAULT_IS_DDMMYY_DATE_FORMAT = false
 const LSS_KEY_IS_24H_TIME_FORMAT* = "is_24h_time_format"
 const DEFAULT_IS_24H_TIME_FORMAT = false
+const LSS_KEY_USER_DECLINED_BACKUP_BUNNER* = "userDeclinedBackupBanner"
```

Comment:
Typo B**A**NNER?? In both constants!

Reply:
Could make it a Rabbits cohort easter egg and call it `BUNNY` instead